test/search.cpp

```diff
@@ -245,4 +245,44 @@ TEST(Search, geoQuery)
   auto result = search.getResults(0, 1);
   ASSERT_EQ(result.begin().getTitle(), "Test Article");
 }
+
+TEST(Search, accents)
+{
+  TempZimArchive tza("testZim");
+
+  zim::writer::Creator creator;
+  creator.configIndexing(true, "nostem");
+  creator.startZimCreation(tza.getPath());
+  creator.addItem(std::make_shared<TestItem>("path0", "text/html", "Test Article0", "This is a tèst articlé. temp0"));
+  creator.addItem(std::make_shared<TestItem>("path1", "text/html", "Test Article1", "This is another test article. For article1."));
```

Comment:
If you follow my advice in the main comment of this review then by moving the accents into the second article the effect of the fix will be more profound).